File: backend/src/main/java/ProgramaClip/ClipMoneyAPI/entidad/Operacion.java
package ProgramaClip.ClipMoneyAPI.entidad;

import java.sql.Timestamp;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.ManyToOne;

import com.sun.istack.NotNull;

import lombok.Data;

@Entity
@Data
public class Operacion {
	@Id
	@GeneratedValue
	private Long id;
	@NotNull
	private Float monto;
	@NotNull
	private Timestamp fechaHora;
	@ManyToOne
	private Cuenta cuentaOrigen;
	@ManyToOne
	@NotNull
	private Cuenta cuentaDestino;
}

File: Models/Operacion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace proyecto_clip_money_e_proyecto_clip.Models
{
    public class Operacion
    {
        private long id;
        private float monto;
        private DateTime fechaHora;
        private Cuenta cuentaOrigen;
        private Cuenta cuentaDestino;

        public Operacion(long id, float monto, DateTime fechaHora, Cuenta cuentaOrigen, Cuenta cuentaDestino)
        {
            this.Id = id;
            this.Monto = monto;
            this.FechaHora = fechaHora;
            this.CuentaOrigen = cuentaOrigen;
            this.CuentaDestino = cuentaDestino;
        }

        public long Id { get => id; set => id = value; }
        public float Monto { get => monto; set => monto = value; }
        public DateTime FechaHora { get => fechaHora; set => fechaHora = value; }
        public Cuenta CuentaOrigen { get => cuentaOrigen; set => cuentaOrigen = value; }
        public Cuenta CuentaDestino { get => cuentaDestino; set => cuentaDestino = value; }
    }
}